pof/src/types.rs

```diff
@@ -1625,6 +1625,16 @@ impl Model {
         (out_vec, out_idx)
     }
 
+    pub fn do_for_recursive_subobj_children(&self, id: ObjectId, f: &mut impl FnMut(&SubObject)) {
+        f(&self.sub_objects[id]);
+
+        let children: Vec<_> = self.sub_objects[id].children().map(|id| *id).collect();
+        for child_id in children {
```

Comment:
```suggestion
        for &child_id in self.sub_objects[id].children() {
```
unnecessary collect

Reply:
Remnant from when this used to give mutable access until I realized I didn't need it.